atomic_dbus.py

```diff
@@ -536,14 +537,16 @@ def TrustDefaultPolicy(self, default_policy):
     # atomic uninstall section
     # The Uninstall method will uninstall the specified image
     @slip.dbus.polkit.require_auth("org.atomic.readwrite")
-    @dbus.service.method("org.atomic", in_signature='ssbas', out_signature='')
-    def Uninstall(self, image, name, force, extra_args):
+    @dbus.service.method("org.atomic", in_signature='ssbsas', out_signature='')
+    def Uninstall(self, image, name=None, force=False, storage=None, extra_args=None):
         i = Uninstall()
         args = self.Args()
         args.image = image
         args.name = name
         args.force = force
-        args.args = extra_args
+        args.storage = storage
+        args.extra_args = [] if not extra_args else extra_args
+        assert(isinstance(args.extra_args, list))
```

Comment:
Yeah it is enforced by the signature. However, given it can be None if not provided, I added the assert.  But I think the line prior enforces that.  So removing...